src/sections/Pricing/review-slider.js

```diff
@@ -57,6 +57,7 @@ const settings = {
 };
 
 const ReviewsWrapper = styled.div`
+max-width: 100%;
```

Comment:
@Shivam-AfA is this change intentional?

Reply:
Yes. This was intentional, else it was overflowing. This doesn't affect any other pages which the review slider is used.